src/main/groovy/org/shipkit/internal/util/EnvVariables.java

```diff
@@ -1,10 +1,29 @@
 package org.shipkit.internal.util;
 
+import org.shipkit.internal.gradle.util.StringUtil;
+
 /**
- * wrapper for System.getenv, so that we can mock it easily in classes where we use it
+ * Utility wrapper for {@link System#getenv(String)}.
+ * Mostly for testing mocking, but also hosts one extra useful method.
  */
 public class EnvVariables {
+
     public String getenv(String name) {
         return System.getenv(name);
     }
+
+    /**
+     * Returns env variable or null if the env variable is empty
+     */
+    public String getNonEmptyEnv(String name) {
+        String value = System.getenv(name);
+        return getNonEmpty(value);
+    }
+
+    static String getNonEmpty(String value) {
```

Comment:
I'm ok either way. This method is not public - it is intended to help testing without actually depending on env variables.